src/contractTest/java/uk/gov/hmcts/reform/iacaseapi/FeesRegisterConsumerTest.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.hmcts.reform.iacaseapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.Pact;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.model.RequestResponsePact;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Maps;
+import io.restassured.RestAssured;
+import io.restassured.config.EncoderConfig;
+import io.restassured.parsing.Parser;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import net.serenitybdd.rest.SerenityRest;
+import org.json.JSONException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.fee.FeeResponse;
+
+@Slf4j
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(SpringExtension.class)
+public class FeesRegisterConsumerTest {
+
```

Comment:
Didn't really need to do this now, but nice if we have it.  The Pact publisher does not work yet in the pipeline so we will have to do that part later.